lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -4641,6 +4641,104 @@ LogicalResult AtenNormScalarOp::verify() {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// AtenRenormOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult AtenRenormOp::verify() {
+
+  auto selfType = cast<BaseTensorType>(getSelf().getType());
+
+  if (!selfType.hasDtype() || !selfType.hasSizes()) {
+    return success();
```

Comment:
Should this be a failure?

Reply:
Recently, I read in one of the PRs that one of the reviewers mentioned that verify() only checks the semantics of the operation itself, such as examining aspects like value ranges, whether dimensions are within the range, whether parameters are of correct types, etc. However, HasDtype and HasSizes are framework matters, meaning that for some reason, information about the data type and sizes of your tensor is not provided, even though all requirements of the operation are met when writing the Python command. In that case, reviewer said that you can simply return success() from verify(), and in the lowering (DecomposeComplexOps.cpp in my case) return failure if necessary.